src/popup/components/Modal.vue

```diff
@@ -258,6 +260,16 @@ export default defineComponent({
     }
   }
 
+  &.transparent {
+    backdrop-filter: blur(10px);
+    background-color: rgba(variables.$color-black, 0.8);
```

Comment:
Is this really necessary as the default value is the same but the opacity is `0.7` instead of `0.8`?

Reply:
According to the design it's 0.8🤷‍♂️ 
@onvisions What do you think?

Reply:
We can leave the default value then. Let's make it 0.7.